src/OpenTelemetry.Api/Trace/ActivityExtensions.cs

```diff
@@ -25,9 +25,14 @@ public static class ActivityExtensions
     /// This extension provides a workaround to store Status as special tags with key name of otel.status_code and otel.status_description.
     /// Read more about SetStatus here https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#set-status.
     /// </summary>
+    /// <remarks>
```

Comment:
I think summary should be cleanup as well, as suggested [here](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4864#discussion_r1330430968).

Reply:
Done